core/src/main/scala/org/genivi/sota/core/PackagesResource.scala

```diff
@@ -113,7 +113,6 @@ class PackagesResource(resolver: ExternalResolverClient,
                      signature: Option[String],
                      file: Source[ByteString, Any]): Future[StatusCode] = {
       val packageCreateF = for {
-        _ <- resolver.putPackage(ns, pid, description, vendor)
```

Comment:
This was the only use of `resolver` so you can kill it off in the constructor as well.